### PR TITLE
Improve authn visuals

### PR DESCRIPTION
**Pull request by feedmeapples**

### CODE REVIEW

File: client/routes/signin.vue [hunk 1]
```diff
@@ -1,61 +1,113 @@
...
-      <div v-else>
-        <button class="close icon icon_delete" @click="oidc">
-          Sign in with SSO
+      <div class="vert-space-4">
+        <button
+          class="close icon icon_delete"
+          @click="logout"
+          v-if="currentUser"
+        >
+          Sing Out
```
Comment (sw-yx) [inline client/routes/signin.vue:30]:
```suggestion
          Sign Out
```

File: client/routes/signin.vue [hunk 2]
```diff
@@ -1,61 +1,113 @@
...
-          <dd>{{ currentUser.name }}</dd>
-          <dt>Email</dt>
-          <dd>{{ currentUser.email }}</dd>
+  <section class="signin">
+    <div class="signin-form">
+      <img
+        :src="
+          currentUser
+            ? currentUser.picture
+            : 'https://seeklogo.com/images/S/snapchat-logo-47531E7AE8-seeklogo.com.png'
```
Comment (sw-yx) [inline client/routes/signin.vue:8]: can we use the temporal logo here as the placeholder?

**Review by sw-yx — APPROVED** [review]
lgtm, left one small nit you may want to address before you merge

File: client/routes/signin.vue (hunk 2 above)
Comment (feedmeapples) [reply client/routes/signin.vue:8]: oops

File: client/App.vue [hunk 3]
```diff
@@ -181,6 +190,15 @@ export default {
...
         <span>{{ $route.params.taskQueue }}</span>
       </div>
+      <div v-if="currentUser" class="user">
+        <img
+          :src="currentUser.picture"
+          width="200"
+          alt="user pic"
+          class="avatar"
+        />
+        <span class="name">{{ currentUser.name }}</span>
```
Comment (sw-yx) [inline client/App.vue:200]:
```suggestion
        <a href="/signin"><span class="name">{{ currentUser.name }}</span></a>
```

can also put the a tag around the img, but then the CSS would need to change

Comment (feedmeapples) [reply client/App.vue:200]: updated

**Pull request merged**